### ISSUE TITLE
(Regression) bug in athena.read_sql_query

### ISSUE THREAD

**Issue by JPFrancoia** [awslabs/aws-data-wrangler#239]
I believe I found a bug in  `athena.read_sql_query`, for awswrangler 1.1.2 (the latest version at the time of this bug report). awswrangler is installed through Poetry.

**Describe the bug**

The following query fails:

```python
max_date_query = wr.athena.read_sql_query(sql=f"select max(date) from my_awesome_table", database="data_dev")
```

The error I get is:

```
QueryFailed: SYNTAX_ERROR: line 1:1: Column name not specified at position 1. You may need to manually clean the data at location 's3://aws-athena-query-results-12345-eu-west-1/tables/tralalalalalalalalalala' before retrying. Athena will not delete data in your account.
```

**To Reproduce**

Directly on Athena, on the AWS console, running this query succeeds:

```
select max(date) from my_awesome_table
```

and returns the date I expect.

Running this query locally fails with the error mentioned above:

```python
max_date_query = wr.athena.read_sql_query(sql=f"select max(date) from my_awesome_table", database="data_dev")
```

If I quickly switch back to awswrangler 0.3.2 and run this:

```python
query = wr.athena.query("select max(date) from my_awesome_table", database="data_dev")
next(query)
```

I get the date I expect.

It's a simple select query so I'm almost sure I didn't do anything wrong (but I won't bet my life on it haha).

Could you give me a hand please?

**Comment by igorborgest** [comment]
Hey @JPFrancoia! Thanks for keep helping us!

This behaviour is inherent from the `ctas_approach=True` where the data will temporary stage on s3 as Parquet files and requires that all result columns has deterministic names and types.

In your case the column `name` is missing.
In this other case (https://github.com/awslabs/aws-data-wrangler/issues/216) the type was missing for example.

That said, I see two options for you:

1 - You can adapt your query to give a name for this columns:

```sql
max_date_query = wr.athena.read_sql_query(
    sql=f"select max(date) as max_date from my_awesome_table",
    database="data_dev"
)
```

2 - Work with the regular Athena's result format (CSV) (As it used to be the default before Wrangler's version `1.0`):

```sql
max_date_query = wr.athena.read_sql_query(
    sql=f"select max(date) from my_awesome_table",
    database="data_dev",
    ctas_approach=False
)
```

**Comment by JPFrancoia** [comment]
It works, thanks! I went for option 1. Sorry I labeled it as a bug.

How can we prevent the future users to fall in this trap?

**Comment by igorborgest** [comment]
I've just add more user-friendly exceptions in the commit above. What you think?

It's already available through:

`pip install git+https://github.com/awslabs/aws-data-wrangler.git@dev`

**Comment by JPFrancoia** [comment]
I just tested with the same query, I indeed triggered the error message specific to my case. It's much more informative, thank you.

It's a very polite error message ;)

**Comment by igorborgest** [comment]
Thank @JPFrancoia! It will be available on version `1.2.0`.

**Comment by igorborgest** [comment]
Released on version [`1.2.0`](https://github.com/awslabs/aws-data-wrangler/releases/tag/1.2.0)
